src/lib.rs

```diff
@@ -207,7 +199,7 @@ impl IntoRawFd for VsockListener {
 
 impl Drop for VsockListener {
     fn drop(&mut self) {
-        unsafe { close(self.socket) };
+        _ = close(self.socket);
```

Comment:
This syntax is unstable on lower rust versions like 1.52, I think let _ is better in compatibility

Reply:
Done.